include/deal.II/matrix_free/operators.h

```diff
@@ -958,7 +958,7 @@ namespace MatrixFreeOperators
                     const LinearAlgebra::distributed::Vector<Number> &src,
                     const std::pair<unsigned int,unsigned int>  &cell_range) const
   {
-    FEEvaluation<dim, fe_degree, fe_degree+1, n_components, Number> phi(data);
+    FEEvaluation<dim, fe_degree, fe_degree+2, n_components, Number> phi(data);
```

Comment:
Correct, VectorTools::project is used with QGauss<dim>(fe.degree+2) in all the (previous) tests and the examples. This makes a difference for `numerics/project_*_03` and `tests/project_01_curved_boundary`.